platform/admin/core/admin-core-logviewer/src/test/js/uniq.js

```diff
@@ -21,7 +21,7 @@ const one = { message: '1', timestamp: 1 }
 const two = { message: '2', timestamp: 2 }
 
 test('old entry (duplicate)', function (t) {
-  t.timeoutAfter(25)
+  t.timeoutAfter(3000)
```

Comment:
Are those times in milliseconds or seconds? If the latter, `3,000` might be a little overkill.